matchzoo/models/cdssm_model.py

```diff
@@ -0,0 +1,95 @@
+"""An implementation of CDSSM (CLSM) model."""
+
+from matchzoo import engine
+
+from keras import Model
+from keras.layers import Input, Conv1D, MaxPooling1D, Dot, Dense, Flatten
+
+
+class CDSSMModel(engine.BaseModel):
+    """
+    Convolutional deep structured semantic model.
```

Comment:
We should add CDSSM paper name in the comment.
Learning Semantic Representations Using Convolutional Neural Networks for Web Search
A Latent Semantic Model with Convolutional-Pooling Structure for Information Retrieval

Reply:
okay